includes/data-port/models/class-sensei-import-course-model.php

```diff
@@ -68,6 +68,7 @@ public function sync_post() {
 
 		$this->set_post_id( $post_id );
 		$this->store_import_id();
+		$this->add_warnings_to_job();
```

Comment:
We'll probably want to do this later on to catch the warnings that are added below (unless I'm missing something?).

Reply:
Yep! Good catch! I'll continue working on this now and update this part. Soon I send the commit here.

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/3443/commits/89a0fea4e84927f35a419bf5b105fda9471c2a8c